subsys/net/lib/lwm2m/lwm2m_message_handling.c

```diff
@@ -2879,24 +2879,37 @@ static int do_send_reply_cb(const struct coap_packet *response, struct coap_repl
 			    const struct sockaddr *from)
 {
 	uint8_t code;
+	struct lwm2m_message *msg = NULL;
```

Comment:
`msg` could be properly initialized rightaway?

Reply:
Done